### ISSUE TITLE
[SRS Review] Functional & Nonfunctional Requirements

### ISSUE THREAD

**Issue by hananezlitni** [whitere123/CAS741_REW#7]
**Section 6: Requirements**

**Functional Requirements:**

- A common standard is phrasing requirements using the verbs "shall" (for mandatory requirements) and "should" (for optional/recommended requirements).. So, it's  better to say "SpecSearch shall take in ..." and so on for the rest of the requirements. The following link provides more details about phrasing requirements: https://softwareengineering.stackexchange.com/a/185871 

- **"Rin" requirement:** the commas separating the input symbols and the sentence list are a little confusing, so you might want to add parentheses to separate the symbols from the list in the sentence, like this: "... conserved quantities (d, g), wave parameters (c, w) and ..."

- **"Rfind" requirement:** the phrase "certain resolution" seems ambiguous, particularly the word "certain", because it can be interpreted differently from a person to another. Can you specify what "certain" can be? Maybe a range of values, or something that the user inputs/selects? 

**Nonfunctional Requirements:**

- The nonfunctional requirements should be measurable (using numbers or other standards), so they can be verified in a later development phase. I have 2 suggestions:

  - NFR2 seems to be related to the "usability" quality. Could you maybe rephrase it so you can add a measurement? Like the ease of use, learning curve, time it takes to learn to use the tool, etc., or

  - Add requirements related to other qualities that can be important for your tool (such as performance, correctness, etc.) and measure them.

**Comment by whitere123** [comment]
Rin - I made the suggested change. 
Rfind - I removed the term resolution entirely. This term is related to the specific numerical algorithm that I intend to use. The SRS should not be this specific.  

I also added more detail to the nonfunctional requirements.
